### PR TITLE
Add updated information for ITK

### PR DESCRIPTION
**Pull request by dzenanz**

### CODE REVIEW

File: data/progress_overwrite.yml [hunk 1]
```diff
@@ -39,6 +39,10 @@ ports:
 - name: vtk
   status: ⚠️
   tracking_issue: "https://gitlab.kitware.com/vtk/vtk/-/issues/19337"
+- name: itk
+  import_statement: itk
+  help_wanted: ✅
```
Comment (kelteseth) [inline data/progress_overwrite.yml:44]:
```suggestion
  status: ✅
```

Comment (kelteseth) [reply data/progress_overwrite.yml:44]: Also, since when does itk feature modules? We need to for the estimated finish

Comment (dzenanz) [reply data/progress_overwrite.yml:44]: We want to have modules, but we don't have a firm plan, mostly due to lack of resources. How about 3 years from now (estimate)?

Comment (kelteseth) [reply data/progress_overwrite.yml:44]: Ah wait so itk does not (yet) support modules? I was confused by the `import_statement: itk` and I figured that it already has modules support. Then my suggestion from earlier does not make sense. Just remove the import_statement. We'll add it when it does support modules

**Comment by kelteseth** [comment]
Perfect, thanks 💪

**Pull request merged**